### PR TITLE
fix/circle-description-bug

### PR DESCRIPTION
**Pull request by ElkLeung**
原先測試只有 desktop chrome 能正常顯示 description，
desktop safari、mobile safari、mobile chrome 皆無法顯示。

現有 line-clamp 屬性的結構為 section > div > p，
理想狀態應為 section > p，第二層的 div 是讓顯示出錯的主因，
但 div 的高度是用來判斷收合狀態與按鈕呈現。

折衷做法是保留 div > p 並讓它隱形（只用來判定），
並另產生 p 作為主要內容。

### CODE REVIEW

File: src/components/Expandable/index.tsx [hunk 1]
```diff
@@ -41,7 +41,18 @@ export const Expandable: React.FC<ExpandableProps> = ({
...
+        style={{
+          position: 'absolute',
+          visibility: 'hidden',
+          opacity: 0,
+          pointerEvents: 'none',
+        }}
+      >
+        {children}
+      </div>
+      {children}
```
Comment (tx0c) [inline src/components/Expandable/index.tsx:55]: it seems to be duplicate of `{children}` ?

**Comment by tx0c** [comment]
vercel build has updated to `npm run vercel-build`, which exists in develop branch, I'm clicking this rebase for you

image

File: src/components/Expandable/index.tsx (hunk 1 above)
Comment (ElkLeung) [reply src/components/Expandable/index.tsx:55]: 對這邊是重複的，上面有詳述這個問題 ↑↑↑

`{ children } `最理想的狀態是：
`<p ref={node}>{ children }</p>`

但目前 children 並非只傳字串會連 tag 一起傳過來，
造成 p 裡頭又包 p 的錯誤。

**Review by robertu7 — CHANGES_REQUESTED** [review]
Left a comment, please review, thanks!

BTW, I saw your commits aren't following the [Commitizen convention](https://github.com/commitizen/cz-cli) ([changelog](https://github.com/thematters/matters-web/releases/tag/v4.0.1) are generated from commits' message), we have added the [git hook](https://github.com/thematters/matters-web/blob/develop/.husky/prepare-commit-msg) on major repos (e.g. `matters-web` and `matters-server`). Is it not working on your local?

image

**Pull request merged**